$NetBSD: patch-cl,v 1.13 2018/04/17 08:03:38 wiz Exp $

- Leave gdbus-2.0/codegen sudirectory to devel/gdbus-codegen package.
- Don't compile tests as they include gdbus-codegen tests.

--- gio/Makefile.am.orig	2018-03-12 16:23:37.000000000 +0000
+++ gio/Makefile.am
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk
 
-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS = 
 
 if OS_UNIX
 SUBDIRS += xdgmime
@@ -246,7 +246,7 @@ platform_libadd += win32/libgiowin32.la
 platform_deps += win32/libgiowin32.la
 endif
 
-SUBDIRS += . tests
+SUBDIRS += .
 
 if HAVE_FAM
 SUBDIRS += fam
